client/my-sites/stats/stats-date-control/stats-date-control-picker-date.tsx

```diff
@@ -16,8 +16,8 @@ const DateControlPickerDate = ( {
 	return (
 		<div className="date-control-picker-date">
 			<div className="stats-date-control-picker-dates__inputs">
-				<TextControl value={ startDate } onChange={ onStartChange } />
-				<TextControl value={ endDate } onChange={ onEndChange } />
+				<TextControl type="date" value={ startDate } onChange={ onStartChange } />
```

Comment:
```suggestion
				<TextControl value={ startDate } onChange={ onStartChange } />
```

Good call to use date input, however `TextControl` does not support the type:

```
[13:22:45 ](https://teamcity.a8c.com/buildConfiguration/calypso_calypso_WebApp_Run_All_Unit_Tests/11017322?hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildProblemsSection=true&showLog=11017322_7612_7612&logView=flowAware)
    client/my-sites/stats/stats-date-control/stats-date-control-picker-date.tsx(19,18): error TS2322: Type '"date"' is not assignable to type '"number" | "search" | "text" | "tel" | "url" | "email" | "password" | undefined'.

[13:22:45 ](https://teamcity.a8c.com/buildConfiguration/calypso_calypso_WebApp_Run_All_Unit_Tests/11017322?hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildProblemsSection=true&showLog=11017322_7613_7613&logView=flowAware)
    client/my-sites/stats/stats-date-control/stats-date-control-picker-date.tsx(20,18): error TS2322: Type '"date"' is not assignable to type '"number" | "search" | "text" | "tel" | "url" | "email" | "password" | undefined'.
```

Let's leave it to another PR.

Reply:
Oops! I was meaning to remove this before pushing the branch. I added it to make entering dates easier/quicker while I was testing things out. Thanks for catching this @kangzj.